polaris-react/src/utilities/css.ts

```diff
@@ -1,5 +1,13 @@
+import type {BreakpointsAlias} from '@shopify/polaris-tokens';
+
 type Falsy = boolean | undefined | null | 0;
 
+export type ResponsiveProp<T> =
+  | T
+  | {
+      [Breakpoint in BreakpointsAlias]?: T;
+    };
```

Comment:
Wondering if this should be exported from tokens instead of this css utils file? Thoughts on moving it?

Reply:
@aaronccasanova would be great to get your thoughts on this